workflows/rnaseq.nf

```diff
@@ -267,7 +277,11 @@ workflow RNASEQ {
         ALIGN_STAR (
             ch_filtered_reads,
             PREPARE_GENOME.out.star_index,
-            PREPARE_GENOME.out.gtf
+            PREPARE_GENOME.out.gtf,
+            params.star_ignore_sjdbgtf,
+            '',
```

Comment:
why have platform if it's always an empty string?

Reply:
I updated the module manually to sync with the nf-core/module as close as possible. This was being used there too. Can remove it but it isn't doing any harm?